pscheduler-archiver-kafka/kafka/archive

```diff
@@ -14,14 +14,13 @@ def archive(json):
     #Do archiving here
 
     data = json["data"]     
-    data = data["data"]
 
     try:
         producer = KafkaProducer(bootstrap_servers=[data["server-address"]])
         producer.send(data["topic"], pscheduler.json_dump(json["result"])).get(timeout=30)
         
     except Exception as e:
-        print(str(e))
+        #print(str(e))
 	result = {
             "succeeded": False,
             "error": "Failed to send message"
```

Comment:
This should include what the exception was so the end user has some idea why it failed.

Reply:
So add it to the error part of the JSON, something like appending "due to error" +  str(e)?

Reply:
The Pythonic way would be `"Failed to send message: %s" % (str(e))`